e2e/tests/visual/defaultPlugins.visual.spec.js

```diff
@@ -0,0 +1,109 @@
+are only meant to run against openmct's app.js started by `npm run start` within the
+`./e2e/playwright-visual.config.js` file.
+
+These should only use functional expect statements to verify assumptions about the state
+in a test and not for functional verification of correctness. Visual tests are not supposed
+to "fail" on assertions. Instead, they should be used to detect changes between builds or branches.
+
+Note: Larger testsuite sizes are OK due to the setup time associated with these tests.
+*/
+
+const { test, expect } = require('../../pluginFixtures');
+const percySnapshot = require('@percy/playwright');
+const { createDomainObjectWithDefaults } = require('../../appActions');
+
+test.describe('Visual - Default', () => {
+    test.beforeEach(async ({ page }) => {
+        //Go to baseURL and Hide Tree
+        await page.goto('./#/browse/mine?hideTree=true', { waitUntil: 'networkidle' });
+    });
+
+    test('Visual - Default Dashboard', async ({ page, theme }) => {
+        // Verify that Create button is actionable
+        await expect(page.locator('button:has-text("Create")')).toBeEnabled();
+
+        // Take a snapshot of the Dashboard
+        await percySnapshot(page, `Default Dashboard (theme: '${theme}')`);
+    });
+
+    test('Visual - Default Condition Set', async ({ page, theme }) => {
+
+        await createDomainObjectWithDefaults(page, {
+            type: 'Condition Set',
+            name: 'Default Condition Set'
+        });
+
+        // Take a snapshot of the newly created Condition Set object
+        await percySnapshot(page, `Default Condition Set (theme: '${theme}')`);
+    });
+
+    test('Visual - Default Condition Widget', async ({ page, theme }) => {
+
+        await createDomainObjectWithDefaults(page, {
+            type: 'Condition Widget',
+            name: 'Default Condition Widget'
+        });
+
+        // Take a snapshot of the newly created Condition Widget object
+        await percySnapshot(page, `Default Condition Widget (theme: '${theme}')`);
+    });
+
+    test('Visual - Sine Wave Generator Form', async ({ page, theme }) => {
+        //Click the Create button
+        await page.click('button:has-text("Create")');
+
+        // Click text=Sine Wave Generator
+        await page.click('text=Sine Wave Generator');
+
+        await percySnapshot(page, `Default Sine Wave Generator Form (theme: '${theme}')`);
+
+        await page.locator('.field.control.l-input-sm input').first().click();
+        await page.locator('.field.control.l-input-sm input').first().fill('');
+
+        // Validate red x mark
+        await percySnapshot(page, `removed amplitude property value (theme: '${theme}')`);
+    });
+
+    test('Visual - Display Layout Icon is correct in Create Menu', async ({ page, theme }) => {
+        // Click the Create button
+        await page.click('button:has-text("Create")');
+
+        // Hover on Display Layout option.
+        await page.locator('text=Display Layout').hover();
```

Comment:
Need to replace this `text=` selector with something more specific to the create menu, this will conflict with the left pane and recent objects (or will it not if left pane is minimized? either way, better to be deterministic here)